app/views/internal/response_templates/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_for [:internal, response_template] do |f| %>
+  <%= f.label :type_of, "Type of Response Template (Mod Comment, Personal Comment, Email, etc)" %>
+  <br>
+  <% type_of_options = [["Mod Comment", "mod_comment"], ["Personal Comment", "personal_comment"], ["Email", "email_reply"], ["Abuse Report Email", "abuse_report_reply"]] %>
+  <%= f.select :type_of, options_for_select(type_of_options, response_template.type_of), html: { required: true } %>
+  <br>
+  <%= f.label :title %>
+  <%= f.text_field :title, value: response_template.title %>
+  <br>
+  <%= f.label :content, "Response template body" %>
+  <br>
+  <%= f.text_area :content %>
+  <br>
+  <%= f.label :content_type, "Content Type (markdown, plain text, HTML, etc)", value: response_template.content %>
+  <br>
+  <% content_type_options = [["Markdown", "body_markdown"], ["Plain Text", "plain_text"], ["HTML", "html"]] %>
+  <%= f.select :content_type, options_for_select(content_type_options, response_template.content_type || "plain_text") %>
+  <br>
+  <%= f.label :user_id, "User ID - Leave blank to allow more than one user to use" %>
+  <br>
+  <%= f.text_field :user_id, value: response_template.user_id %>
+  <br>
+  <br>
+  <%= f.submit %>
+<% end %>
+<div style="margin-bottom: 75px;"></div>
```

Comment:
I'm not sure we should be doing this, we can always add the class `mb-5` to the form tag and bootstrap will take care of it!